draft-ietf-rats-architecture.md

```diff
@@ -330,11 +330,11 @@ a known, good operational state.
 In the Fast IDentity Online (FIDO) protocol {{WebAuthN}}, {{CTAP}}, the device in the user's hand authenticates the human user, whether by biometrics (such as fingerprints), or by PIN and password.
 FIDO authentication puts a large amount of trust in the device compared to typical password authentication because it is the device that verifies the biometric, PIN and password inputs from the user, not the server.
 For the Relying Party to know that the authentication is trustworthy, the Relying Party needs to know that the Authenticator part of the device is trustworthy.
-The FIDO protocol employs remote attestation for this.
+The FIDO protocol employs Remote Attestation Procedures for this.
```

Comment:
I think Laurence's point was this should be reverted since FIDO doesn't use RATS WG output?